cmd/openshift-install/create.go

```diff
@@ -266,6 +266,10 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config, director
 	}, 2*time.Second, apiContext.Done())
 	err = apiContext.Err()
 	if err != nil && err != context.Canceled {
+		err2 := runGatherBootstrapCmd(nil, nil)
```

Comment:
IMO it makes sense to call this after we have decided that waitForBootstrap has failed [here](https://github.com/openshift/installer/pull/1627/files#diff-266f03122200ace2d21f2e449b6a5503R99)

Reply:
Agree, done.

Reply:
> IMO it makes sense to call this after we have decided that `waitForBootstrap` has failed here

Don't we want `create cluster` failures to spit this out too?  I'd expect we want it in `waitForBootstrapComplete` where `create cluster` and `wait-for bootstrap-complete` can share it.

Reply:
the create cluster and wait-for cannot share this anyways because create cluster we are sure that we wouldn't need the `--{bootstrap,control-plane}-address` flags but, wait-for if might need them.

So if it was upto me i would not call this at all and just print something like `call gather bootstrap-failure subcommand` to gather logs

Reply:
I think this makes sense if/when this command can actually gather the logs. Else we are asking them to run one command just to output more commands for them to run. May as well get to the point for now.